cdap-app-templates/cdap-etl/cdap-etl-core/src/main/java/co/cask/cdap/etl/planner/PipelinePlan.java

```diff
@@ -50,6 +52,26 @@ public PipelinePhase getPhase(String name) {
     return phaseConnections;
   }
 
+  /**
+   * @return Conditions along with their phase connections
+   */
+  public Map<String, Map<Boolean, String>> getConditionPhaseConnections() {
```

Comment:
I think it's better to use that ConditionBranches class instead of a `Map<Boolean, String>`